### DISCUSSION TITLE
How to setup the "authors" key in the config file for "twitter:creator" and "article:author" meta tags?

### DISCUSSION THREAD

**Discussion by icy-comet** [adityatelange/hugo-PaperMod#340] [Q&A] [answered]
I checked the template files.

The tags are enclosed this way:

`{{ range .Site.Authors }}
{{ with .twitter -}}
<meta name="twitter:creator" content="@{{ . }}"/>
{{ end -}}
{{ end -}}`

How to setup the "authors" key in the config file? I couldn't figure it out.

No matter how you setup `authors` in the `config` file, the value never gets read and the tags are never rendered in the final html page.

I tried:
```yml
authors:
    Me:
       twitter: twitter handle
```

**Comment by pixelfabs** [comment, marked as answer]
You're not alone. This is what I find out: it was a variable made available in Hugo but was never finished 100%. It was removed from the documentation and a new proposal was made for how to implement it. So, your experience is correct. It just never rendered, no matter how you set it up. It also gave no obvious errors relating to authors. https://discourse.gohugo.io/t/site-author-usage/31459/15

Current PaperMod has it also removed. If you do like to have it, a single author is easy. See link above for some examples.
